### PR TITLE
[OSDOCS-11001]: Migrating HCP resource utilization content from RHACM to OCP

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-11724

Link to docs preview: https://80620--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-override-resource-util.html

QE review:
- [x] QE has approved this change.

**NOTE FOR PEER REVIEWERS:** This PR moves the HCP resource utilization content from the RHACM docs repo to the OCP docs repo. The content has not been changed since it was peer reviewed and approved by the RHACM docs team.

### CODE REVIEW

File: modules/hcp-disable-metrics.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    value: "80"
+  - name: hcThresholdNumber
+    value: "60"
+  - name: disableMetrics <1>
+    value: "true"
+----
++
+<1> The `disableMetrics=true` customized variable disables the metric service monitoring configuration for both new and existing `hypershift-addon` managed cluster add-ons.
+
+. Save the changes.
```
Comment (abrennan89) [inline modules/hcp-disable-metrics.adoc:50]: Maybe update this command instead to apply the changes to the YAML?
```suggestion
. Apply the changes to the configuration specification by running the following command:
+
[source,terminal]
----
$ oc apply -f <filename>.yaml
----
```

Follow up question that may be out of scope here, but do any services or clusters etc need to be restarted for the changes to take effect?

File: modules/hcp-override.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-override-resource-util.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-override_{context}"]
+= Overriding resource utilization measurements for a hosted cluster
+
+You can override the resource utilization measurements based on the type and pace of your cluster workload. Complete the following steps:
```
Comment (abrennan89) [inline modules/hcp-override.adoc:9]:
```suggestion
You can override resource utilization measurements based on the type and pace of your cluster workload.
```
Suggestion to remove the second lead-in sentence which seems unnecessary

**Review by abrennan89 — COMMENTED** [review]
Some comments, mostly just suggestions 🙂 nice work

File: modules/hcp-disable-metrics.adoc (hunk 1 above)
Comment (lahinson) [reply modules/hcp-disable-metrics.adoc:50]: Good question! I don't think so. Engineering and QE approved this text, and they didn't mention that anything else needed to be restarted for the changes to take effect.

**Comment by lahinson** [comment]
Thanks for the suggestions, @abrennan89! I'm incorporating them now :)

**Comment by abrennan89** [comment]
@lahinson I think this might need to be cherrypicked to the 4.17 branch also? 🙂

**Pull request merged**